src/structures/APIMessage.js

```diff
@@ -176,6 +176,11 @@ class APIMessage {
       flags = this.options.flags != null ? new MessageFlags(this.options.flags).bitfield : this.target.flags.bitfield;
     }
 
+    const allowed_mentions =
```

Comment:
We do not use snake case identifiers:
```suggestion
    const allowedMentions =
```